osbs/build/build_request.py

```diff
@@ -732,6 +737,11 @@ def render(self, validate=True):
             self.spec.validate()
         super(ProductionBuild, self).render()
 
+        repo_name = git_repo_humanish_part_from_uri(self.spec.git_uri.value)
+        # What about forked repos?
```

Comment:
I think deferring this use case is valid at this point, especially because the branch name is taken into account. As a side note, we could always hash the URI and branch together and use that as a label, or maybe even as the build config name.